src/index.js

```diff
@@ -250,7 +250,7 @@ Inspector.prototype = {
       this.sceneEl.setAttribute('aframe-inspector-removed-embedded');
     }
 
-    document.body.classList.add('aframe-inspector-opened');
+    document.body.classList.add('aframe-inspector-isOpened');
```

Comment:
why change to `isOpened`? This may break core ui in core 3dstreet repo that assumes the old 'aframe-inspector-opened' syntax.

```
 document.body.classList.add('aframe-inspector-opened');
    document.body.classList.add('aframe-inspector-isOpened');
```